src/main/java/uk/ac/ebi/embl/gff3tools/utils/ConversionUtils.java

```diff
@@ -43,26 +46,49 @@ public static Map<String, Set<String>> getFeatureRelationMap() {
         return INSTANCE.featureRelations;
     }
 
-    public static Map<String, ConversionEntry> getGFF32FFFeatureMap() {
-        return INSTANCE.gff32ff;
-    }
-
     public static Map<String, String> getGFF32FFQualifierMap() {
         return INSTANCE.gff32ffQualifiers;
     }
 
+    // Returns the EMBL feature name for an SOTerm
+    public static ConversionEntry getINSDCFeatureForSOTerm(String SOTerm) {
+        ConversionEntry conversionEntry = INSTANCE.gff32ff.get(SOTerm);
+        if (conversionEntry == null) {
+            Stream<String> parents = INSTANCE.ontologyClient.getParents(SOTerm);
```

Comment:
We could log when we get values using ontologyClient. Later, we can analyse the logs to identify the usage of ontologyClient